tests/common/widgets/hits-per-page/options.ts

```diff
@@ -0,0 +1,175 @@
+import { createSearchClient } from '@instantsearch/mocks';
+import { normalizeSnapshot, wait } from '@instantsearch/testutils';
+import { screen } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+
+import type { HitsPerPageWidgetSetup } from '.';
+import type { TestOptions } from '../../common';
+
+export function createOptionsTests(
+  setup: HitsPerPageWidgetSetup,
+  { act }: Required<TestOptions>
+) {
+  describe('options', () => {
+    test('renders with required props', async () => {
+      const searchClient = createSearchClient({});
+
+      await setup({
+        instantSearchOptions: {
+          indexName: 'indexName',
+          searchClient,
+        },
+        widgetParams: {
+          items: [
+            { label: '10', value: 10, default: true },
```

Comment:
we should also have a test where default isn't the first item

Reply:
Done in 0a4413eee.